src/mesh_motion/MotionPulsatingSphere.C

```diff
@@ -30,8 +40,9 @@ void MotionPulsatingSphere::load(const YAML::Node& node)
 
   get_if_present(node, "frequency", frequency_, frequency_);
 
-  origin_ = node["origin"].as<ThreeDVecType>();
-  assert(origin_.size() == threeDVecSize);
+  // get origin based on if it was defined
+  if( node["centroid"] )
```

Comment:
Is this name change from `origin` to `centroid` in the input file intentional? Is this consistent with other mesh motion types?

Reply:
This was an error on my end when I committed it. It has always been `centroid` since #77. Here I made it consistent with the other motions.